notifications.yaml

```diff
@@ -0,0 +1,220 @@
+swagger: "2.0"
+info: 
+  version: "0.0.1"
+  title: "Notification API"
+  description: "Notification API."
+host: "localhost"
+basePath: "/api/v1"
+schemes: 
+  - "https"
+paths: 
+  /Messages/{id}: 
+    parameters: 
+      - 
+        name: "id"
```

Comment:
I think it's worth being more specific with the name, to avoid confusion and ambiguity, like `notificationId`